pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -651,6 +651,14 @@ func (c *ConfigMapUnpacker) ensureJob(cmRef *corev1.ObjectReference, bundlePath
 
 		return
 	}
+	// Cleanup old unpacking job and retry
```

Comment:
If we don't care about persisting the failed job at all, this can be simplified to deleting the job immediately after failure and waiting for the next resolver run.

Reply:
I think we should persist the failed job - we need a debug trail of some sort